src/index.json

```diff
@@ -26996,6 +26996,77 @@
                     "version": "2.5.0"
                 },
                 "sha256Digest": "66c2c3dd71040be0ba304b80e25e8f5f2196caa3970e3c7e15d260243dde31e7"
+            },
+            {
+                "downloadUrl": "https://azuremlsdktestpypi.blob.core.windows.net/wheels/sdk-cli-v2-public/ml-2.6.1-py3-none-any.whl",
```

Comment:
Please change the version from 2.6.1 to 2.6.0

Reply:
Done.